[Bug][Testing] Broken `test_loss_functions.py`

### Bug Explanation

A number of open PRs are being affected by `test_loss_functions.py`, which is failing on the CI with the following error message ((example job)[https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736]):

```
_ ERROR collecting ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py _
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:62)
ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py:203: in <module>
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:63)
    dtype_and_pos_weight=helpers.array_or_none(
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:64)
E   AttributeError: module 'ivy_tests.test_ivy.helpers' has no attribute 'array_or_none'
(https://github.com/unifyai/ivy/actions/runs/3853660784/jobs/6566851736#step:6:65)
_ ERROR collecting ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py _
```

The problematic `helpers.array_or_none()` function was deleted in commit c8d0524e83f1b09ee8c9b3ec5d4de223e50ea8c4.

### Steps to Reproduce Bug

CI logs

### Environment

N/A

### Ivy Version

master

### Backend

- [ ] NumPy
- [ ] TensorFlow
- [X] PyTorch
- [ ] JAX

### Device

_No response_

--- a/ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py
+++ b/ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py
@@ -200,16 +200,19 @@ def test_torch_binary_cross_entropy(
     size_average=st.booleans(),
     reduce=st.booleans(),
     reduction=st.sampled_from(["mean", "none", "sum", None]),
-    dtype_and_pos_weight=helpers.array_or_none(
-        array_dtype="float",
-        min_value=0,
-        max_value=10,
-        allow_inf=False,
-        exclude_min=True,
-        exclude_max=True,
-        min_num_dims=1,
-        max_num_dims=1,
-        min_dim_size=2,
+    dtype_and_pos_weight=st.one_of(
+        helpers.dtype_and_values(
+            array_dtype="float",
+            min_value=0,
+            max_value=10,
+            allow_inf=False,
+            exclude_min=True,
+            exclude_max=True,
+            min_num_dims=1,
+            max_num_dims=1,
+            min_dim_size=2,
+        ),
+        st.just([None], [None]),
     ),
 )
 def test_torch_binary_cross_entropy_with_logits(

--- a/ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py
+++ b/ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py
@@ -202,7 +202,7 @@ def test_torch_binary_cross_entropy(
     reduction=st.sampled_from(["mean", "none", "sum", None]),
     dtype_and_pos_weight=st.one_of(
         helpers.dtype_and_values(
-            array_dtype="float",
+            available_dtypes=helpers.get_dtypes("float"),
             min_value=0,
             max_value=10,
             allow_inf=False,
@@ -212,7 +212,7 @@ def test_torch_binary_cross_entropy(
             max_num_dims=1,
             min_dim_size=2,
         ),
-        st.just([None], [None]),
+        st.just([[None], [None]]),
     ),
 )
 def test_torch_binary_cross_entropy_with_logits(

